onnxruntime/core/providers/armnn/activation/activations.cc

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) 2020, NXP Semiconductor, Inc. All rights reserved.
+// Licensed under the MIT License
+
+#ifdef _WIN32
+#pragma warning(disable : 4244)
+#endif
+#include <thread>
+#include <mutex>
+
+#include "core/providers/armnn/armnn_common.h"
+#include "core/providers/armnn/activation/activations.h"
+#include "core/providers/armnn/armnn_fwd.h"
+
+#include "armnn/ArmNN.hpp"
```

Comment:
Wouldn't we need to set CPATH to the cloned equivalent of `https://github.com/ARM-software/armnn/tree/branches/armnn_20_02/include` for this to work? If that's the case, we should add that information to BUILD.md.

Reply:
ArmNN EP was validated with NXP i.MX SDK toolchain which includes the ArmNN header and librariesas in the rootfs, similar like we did for initial ACL EP commit. We will update the patch description to reflect this. The EP should be compatible with an standalone ArmNN build but we are not going to cover it with this commit.